.env.example

```diff
@@ -17,6 +17,7 @@ CKAN_SITE_URL=http://localhost:5500
 CKAN__CORS__ORIGIN_WHITELIST=http://localhost:4200
 CKAN_PORT=5000
 CKAN___BEAKER__SESSION__SECRET=CHANGE_ME
+CKAN__AUTH__USER_CREATE_ORGANIZATIONS=false
```

Comment:
https://github.com/GenomicDataInfrastructure/gdi-userportal-ckan-deployment  Can you maybe change it here also?

Reply:
Thanks. Here it is
https://github.com/GenomicDataInfrastructure/gdi-userportal-ckan-deployment/pull/10